yarn-project/noir-contracts/contracts/inclusion_proofs_contract/src/main.nr

```diff
@@ -78,7 +23,7 @@ contract InclusionProofs {
     // docs:start:create_note
     // Creates a value note owned by `owner`.
     #[aztec(private)]
-    fn create_note(owner: AztecAddress, value: Field) {
+    fn create_new_note(owner: AztecAddress, value: Field) {
```

Comment:
Is this because it clashes with a newly imported `create_note` function? If so that's unfortunate that we're polluting the namespace in this way, since the user has no way of knowing which functions are being brought it.

Perhaps it'd be a good idea to have them inside an e.g. `aznr` namespace? And do `aznr::get_note`.

Reply:
+1 - otherwise we are adding a bunch of "keywords" without people realising.

Reply:
Names like `create_note` and `destroy_note` were the main ones I was thinking should not be in the prelude and we could include others we feel would lead to common clashes. I will look at moving these to a namespace.